Index: mod_defensible-1.5/Makefile.am
===================================================================
--- mod_defensible-1.5.orig/Makefile.am
+++ mod_defensible-1.5/Makefile.am
@@ -8,8 +8,8 @@ MODULESOURCES = mod_defensible.c
 lib_LTLIBRARIES = mod_defensible.la
 
 mod_defensible_la_SOURCES = $(MODULESOURCES)
-mod_defensible_la_LDFLAGS = -module `${APXS2} -q LDFLAGS_SHLIB`
-mod_defensible_la_CPPFLAGS = -I`${APXS2} -q INCLUDEDIR`
+mod_defensible_la_LDFLAGS = -module -avoid-version
+mod_defensible_la_CPPFLAGS = -I`${APXS2} -q INCLUDEDIR` -I`apr-1-config --includedir` `apr-1-config --cppflags`
 mod_defensible_la_CFLAGS = -W -Wall `${APXS2} -q CFLAGS CFLAGS_SHLIB`
 CC = `${APXS2} -q CC`
 
